specification/devcenter/data-plane/Microsoft.DevCenter/stable/2023-04-01/devbox.json

```diff
@@ -1101,7 +1101,7 @@
           "$ref": "#/definitions/HardwareProfile"
         },
         "hibernateSupport": {
-          "description": "Indicates whether hibernate is enabled/disabled or unknown.",
+          "description": "Indicates whether hibernate is enabled/disabled or unknown. (Feature in Preview)",
```

Comment:
You cannot mark features in a GA version as "preview".  This defeats the whole purpose of the preview api-version.

Reply:
@mikekistler Sorry, should have given more background in the description for this! Our API is stable and we have no anticipated changes to the API surface. In the background, hibernation relies on a preview feature from compute, and may not meet production SLAs. We want to echo what the compute RP uses to describe the hibernate parameters in their stable API so that that information is available to customers: https://github.com/Azure/azure-rest-api-specs/blob/d0d161eb19d5efd1c3d23d47a5491a53f41cbd7c/specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-03-01/virtualMachine.json#L999 
Is there another way that we should be indicating that the feature is a preview feature with a stable API surface? I looked around but I wasn't able to find docs on this.